src/jaxsim/mujoco/visualizer.py

```diff
@@ -46,7 +46,7 @@ def __init__(
 
         self.fps = fps
         self.frames: list[npt.NDArray] = []
-        self.data: mujoco.MjData | None = None
+        self.data: list[mujoco.MjData] | mujoco.MjData | None = None
```

Comment:
Why is there `data` as init argument, which is not used?
Should we change the type also there?

Reply:
ups, yes got it, thanks!